### PR TITLE
Added include owned option to get fields

### PR DESCRIPTION
**Pull request by HadarLevi556**
Add include_owned option to GET endpoint of fields

### CODE REVIEW

**Comment by Chasky-H** [comment]
These are breaking changes
No can do.

File: endpoints/meta-data.ts [hunk 1]
```diff
@@ -55,13 +55,21 @@ export class SubTypes {
 export class Fields {
     constructor(private service: PapiClient, private type: string, private subtypeid?: string) {}
 
-    async get(): Promise<ApiFieldObject[]>;
     async get(apiName: string): Promise<ApiFieldObject>;
-    async get(apiName?: string): Promise<ApiFieldObject | ApiFieldObject[]> {
+    async get(params?: { include_owned: boolean }): Promise<ApiFieldObject[]>;
+    async get(p?: string | { include_owned: boolean }): Promise<ApiFieldObject | ApiFieldObject[] | undefined> {
```
Comment (Chasky-H) [inline endpoints/meta-data.ts:60]: Why is there undefined in the return value

Comment (Chasky-H) [reply endpoints/meta-data.ts:60]: What about default param for include_owned

**Review by Chasky-H — CHANGES_REQUESTED** [review]

File: entities/api-field-object.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
 export interface ApiFieldObject {
+    InternalID: number;
```
Comment (Chasky-H) [inline entities/api-field-object.ts:2]: Should be optional if it isn't required in post

File: endpoints/meta-data.ts [hunk 3]
```diff
@@ -50,18 +50,30 @@ export class SubTypes {
...
+    addons = new Addons(this.service, this.typeName, this.subtype);
 }
 
 export class Fields {
     constructor(private service: PapiClient, private type: string, private subtypeid?: string) {}
 
-    async get(): Promise<ApiFieldObject[]>;
     async get(apiName: string): Promise<ApiFieldObject>;
-    async get(apiName?: string): Promise<ApiFieldObject | ApiFieldObject[]> {
+    async get(params: { include_owned: boolean }): Promise<ApiFieldObject[]>;
```
Comment (Chasky-H) [inline endpoints/meta-data.ts:61]: Shouldn't this parameter be optional

**Review by Chasky-H — CHANGES_REQUESTED** [review]

**Comment by Chasky-H** [comment]
Something is wrong with the package.json. Why did you install @pepperi-addons/scripts?
Also, you did not bump a version

**Comment by HadarLevi556** [comment]
> Something is wrong with the package.json. Why did you install @pepperi-addons/scripts?
> Also, you did not bump a version

Probably by mistake
I did uninstall and upgrade the version

**Review by Chasky-H — APPROVED** [review]

**Pull request merged**